app/controllers/home_controller.rb

```diff
@@ -3,10 +3,10 @@ class HomeController < ApplicationController
 
   def index
     # rubocop:disable Style/MultilineOperationIndentation
-    @studies = Study.send(@study_scope).
-                     order(updated_at: :desc).
-                     page(params[:page]).
-                     per(10)
+    @studies = get_filtered_studies.
```

Comment:
It seems like this could just use `get_search_results` from the search controller (maybe promoting that to the concern class), since it's basically doing the same thing, just with no search term?